test/app/teams_test.rb

```diff
@@ -91,6 +91,17 @@ def test_user_must_be_manager
     end
   end
 
+  def test_logged_in_user_can_search_teams
+    [
+      [:get, '/teams'],
+      [:get, '/teams/']
+    ].each do |verb, endpoint|
```

Comment:
I personally don't feel strongly about looping. I use it all the time in Go. The one reason I have to not use it with minitest is that once you have an assertion, none of the following test cases are run, so it can obscure the scope of an issue.

I'm fine with either of the approaches suggested.